src/TesApi.Tests/Runner/TaskExecutionScriptingManagerTests.cs

```diff
@@ -90,9 +87,6 @@ public async Task PrepareBatchScript_ValidTask_CreatesAndUploadsExpectedAssetsTo
                     It.IsAny<NodeTaskConversionOptions>(), It.IsAny<CancellationToken>()),
                 Times.Once);
 
-            //creates the script
```

Comment:
I'd like to find a way to still validate that the script builder was invoked exactly once. Maybe move the call to create and invoke the builder to its own method and verify that the method is called? However, I wouldn't hold up this PR to restore this part of the test, so if something easy doesn't present itself, don't do more work on it.